src/Calculator/Common/AlwaysSelectedCollectionView.cs

```diff
@@ -0,0 +1,264 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Interop;
+
+namespace CalculatorApp
+{
+    namespace Common
+    {
+        sealed class AlwaysSelectedCollectionView : Windows.UI.Xaml.DependencyObject, Windows.UI.Xaml.Data.ICollectionView
+        {
+            internal AlwaysSelectedCollectionView(IList source)
+            {
+                m_currentPosition = -1;
+                m_source = source;
+
+                var observable = source as Windows.UI.Xaml.Interop.IBindableObservableVector;
+                if (observable != null)
+                {
+                    observable.VectorChanged += OnSourceBindableVectorChanged;
+                }
+            }
+
+            public bool MoveCurrentTo(object item)
+            {
+                if (item != null)
+                {
+                    int newCurrentPosition = m_source.IndexOf(item);
+                    //bool result = m_source.IndexOf(item, &newCurrentPosition);
```

Comment:
Can we remove this comment since the newCurrentPosition is updated?

Reply:
Removed.